### PR TITLE
2689 turbo mode experiment

### PR DESCRIPTION
**Pull request by Baron-Severin**
## Checklist

- [X] Confirm the **acceptance criteria** is fully satisfied in the issue(s) this PR will close
AC specify that settings Turbo icon is updated, but Turbo is not currently in our settings channel. Other than that, all AC are fulfilled
- [X] Add thorough **tests** or an explanation of why it does not
UI changes only
- [X] Add a **CHANGELOG entry** if applicable
- [X] Add **QA labels** on the associated issue (not this PR; `qa-ready` or `qa-notneeded`)

### CODE REVIEW

File: app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/ToolbarUiController.kt [hunk 1]
```diff
@@ -153,11 +153,12 @@ class ToolbarUiController(
 
             layout.desktopModeButton.isChecked = it.desktopModeChecked
             layout.turboButton.isChecked = it.turboChecked
-            layout.turboButton.contentDescription = layout.context.resources.getString(
-                // TODO known issue: updating the content description doesn't update the hanger until focus navigates away and back again. fix this
+            val turboText = layout.context.resources.getString(
                 turboButtonContent.contentDescriptionId,
                 if (it.turboChecked) "ON" else "OFF"
```
Comment (dnarcese) [inline app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/ToolbarUiController.kt]: This "ON" and "OFF" would need to be localized.

File: app/src/main/res/drawable/ic_etp_disabled.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (dnarcese) [inline app/src/main/res/drawable/ic_etp_disabled.xml]: nit: license

File: app/src/main/res/drawable/ic_etp_enabled.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (dnarcese) [inline app/src/main/res/drawable/ic_etp_enabled.xml]: nit: license

**Comment by dnarcese** [comment]
Also, don't forget to add this experiment to Kinto!

File: app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/ToolbarUiController.kt (hunk 1 above)
Comment (Baron-Severin) [reply app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/ToolbarUiController.kt]: 🤦‍♂ 
I don't know how I missed that.  Thank you for catching it.

**Comment by Baron-Severin** [comment]
> Also, don't forget to add this experiment to Kinto!

Done :+1:

**Pull request merged**